app/build.gradle

```diff
@@ -85,12 +85,21 @@ dependencies {
     implementation "net.e175.klaus:solarpositioning:0.0.9"
     // use pristine greendao instead of our custom version, since our custom jitpack-packaged
     // version contains way too much and our custom patches are in the generator only.
+
+    implementation 'com.twofortyfouram:android-plugin-client-sdk-for-locale:[4.0.3, 5.0['
+    implementation 'com.twofortyfouram:android-plugin-host-sdk-for-locale:[2.0.3,3.0['
+    implementation 'com.twofortyfouram:android-plugin-api-for-locale:[1.0.2,2.0['
```

Comment:
What is com.twofortyfouram:android-plugin-* ? Why is that needed?

Reply:
that has to do with tasker, i will try to remove that ASAP